webapp/src/views/Collection.vue

```diff
@@ -8,14 +8,21 @@
           variant="danger"
           dismissible
         >
-          Elasticsearch server denied access. Please check its access policy.
+          Elasticsearch error. Please check browser and elasticsearch access logs.
         </b-alert>
         <b-alert
           v-model="showDataplaneAlert"
           variant="danger"
           dismissible
         >
-          Failed to connect to dataplane. Please check access control policy in API Gateway.
+          Dataplane Error. Please check browser and dataplane logs.
```

Comment:
Similar to above, except now we aren't even using a resource policy in the API's so this would just be confusing.